roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -14,6 +14,25 @@ assetConfig:
 {% if 'metrics_public_url' in openshift.master %}
   metricsPublicURL: {{ openshift.master.metrics_public_url }}
 {% endif %}
+{% if 'extension_scripts' in openshift.master %}
+  extensionScripts:
+{% for script in openshift.master.extension_scripts %}
```

Comment:
We won't need the loop with `to_padded_yaml` so this would become:

```
{% if 'extension_scripts' in openshift.master %}
  extensionScripts:
{{ openshift.master.extension_scripts | to_padded_yaml(1, 2) }}
{% endif %}
```